lib/src/functions.dart

```diff
@@ -943,3 +950,24 @@ class Sgn extends DefaultFunction {
     throw UnimplementedError('Can not evaluate $name on $type yet.');
   }
 }
+
+class GenericFunction extends DefaultFunction {
+  /// Creates a genetic function with variable number of arguments.
+  ///
+  /// For example, to create min(2,23,4,5,2314,213,5,324):
+  ///
```

Comment:
I think there's an example missing? I added one to the library examples on my branch, see my other comment, so this could be removed.

However this section should have some documentation making the purpose of this class as facility to evaluate expressions via Dart clear (currently that's not mentioned); and outline the major differences to the builtin functions, e.g. no support for differentiation, error potential etc.